src/Tribe/Main.php

```diff
@@ -3828,17 +3836,25 @@ public function isVenue( $postId = null ) {
 		 * @return bool Is it an organizer?
 		 */
 		public function isOrganizer( $postId = null ) {
+			static $is_organizer = [];
+
 			if ( $postId === null || ! is_numeric( $postId ) ) {
 				global $post;
 				if ( isset( $post->ID ) ) {
 					$postId = $post->ID;
 				}
 			}
+
+			// Return if we've already fetched this info.
+			if ( isset( $is_organizer[ $postId ] ) ) {
+				return $is_organizer[ $postId ];
+			}
+
 			if ( isset( $postId ) && get_post_field( 'post_type', $postId ) == Tribe__Events__Organizer::POSTTYPE ) {
-				return true;
+				return $is_organizer[ $postId ] = true;
 			}
 
-			return false;
+			return $is_organizer[ $postId ] = false;
```

Comment:
* :no_entry_sign: Variable "$postId" is not in valid snake_case format, try "$post_id"
* :no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 8760d926f63ebcd45d54c5a20e85edbf</sub>

Reply:
Ignoring.